client/dashboard/components/date-range-picker/date-range-content.tsx

```diff
@@ -94,33 +78,65 @@ export function DateRangeContent( props: DateRangeContentProps ) {
 	};
 
 	const setPreset = ( id: PresetId ) => {
-		const range = computePresetRange( id, today );
+		// For presets, use site timezone, allowing users to access logs from the site's current day
+		let presetToday: Date;
+
+		if ( timezoneString ) {
+			const now = new Date();
+			const siteTime = new Intl.DateTimeFormat( 'en-US', {
+				timeZone: timezoneString,
+				year: 'numeric',
+				month: 'numeric',
+				day: 'numeric',
+			} ).formatToParts( now );
+
+			const year = parseInt( siteTime.find( ( p ) => p.type === 'year' )?.value || '0' );
+			const month = parseInt( siteTime.find( ( p ) => p.type === 'month' )?.value || '0' ) - 1; // Month is 0-indexed
+			const day = parseInt( siteTime.find( ( p ) => p.type === 'day' )?.value || '0' );
+
+			presetToday = startOfDay( new Date( year, month, day ) );
+		} else if ( typeof gmtOffset === 'number' ) {
+			// Use GMT offset if no timezone string
+			const now = new Date();
+			presetToday = startOfDay( addHours( now, gmtOffset ) );
+		} else {
+			// Fallback to UTC (default behavior)
+			const utcNow = new Date();
+			presetToday = startOfDay(
+				new Date( Date.UTC( utcNow.getUTCFullYear(), utcNow.getUTCMonth(), utcNow.getUTCDate() ) )
+			);
+		}
+
+		const range = computePresetRange( id, presetToday );
 		if ( ! range ) {
 			return;
 		}
 		setFromDraft( range.from );
 		setToDraft( range.to );
-		setFromStr( formatYmd( range.from, timezoneString, gmtOffset ) );
-		setToStr( formatYmd( range.to, timezoneString, gmtOffset ) );
+		// Use UTC formatting without timezone conversion for consistency
+		setFromStr( format( range.from, 'yyyy-MM-dd' ) );
+		setToStr( format( range.to, 'yyyy-MM-dd' ) );
```

Comment:
Showing the strings always in UTC seems strange to me. Shouldn't the times always be presented in the site's timezone?